src/go/configgenerator/filtergen/util.go

```diff
@@ -48,19 +46,19 @@ func MethodToSelector(api *apipb.Api, method *apipb.Method) string {
 
 // MethodToCORSSelector gets the corresponding autogenerated CORS selector for
 // an OP API and Method.
-func MethodToCORSSelector(api *apipb.Api, method *apipb.Method) string {
-	return api.GetName() + corsOperationDelimiter + method.GetName()
+func MethodToCORSSelector(api *apipb.Api, method *apipb.Method, CORSOperationDelimiter string) string {
```

Comment:
nit: naming

`corsOperationDelimiter` for local variable use (here)
`CORSOperationDelimiter` for exported struct use